dev/TabView/TabView_themeresources.xaml

```diff
@@ -17,6 +17,21 @@
             <StaticResource x:Key="TabViewItemHeaderForegroundPointerOver"   ResourceKey="SystemControlForegroundBaseHighBrush" />
             <StaticResource x:Key="TabViewItemHeaderForegroundDisabled"      ResourceKey="SystemControlDisabledBaseMediumLowBrush" />
             <StaticResource x:Key="TabViewItemIconForeground"                ResourceKey="SystemControlForegroundBaseMediumBrush" />
+            <StaticResource x:Key="TabViewItemIconForegroundPressed"         ResourceKey="SystemControlForegroundBaseMediumBrush" />
+            <StaticResource x:Key="TabViewItemIconForegroundSelected"        ResourceKey="SystemControlForegroundBaseMediumBrush" />
+            <StaticResource x:Key="TabViewItemIconForegroundPointerOver"     ResourceKey="SystemControlForegroundBaseMediumBrush" />
+            <StaticResource x:Key="TabViewItemIconForegroundDisabled"        ResourceKey="SystemControlForegroundBaseMediumBrush" />
```

Comment:
I think you should probably use a MediumLow color in the disabled state, probably SystemControlDisabledBaseMediumLowBrush

Reply:
I did this, and also tried to unify the brushes between the themes where possible.  Now the 14 brushes that differ in HighContrast theme actually do (intentionally) yield different colors than the brushes from Dark/Light themes.